notebooks/special_linear.ipynb

```diff
@@ -0,0 +1,337 @@
+{
```

Comment:
<p>To make this notebook pass, try adding a Set up section with the code pasted <a href="https://github.com/geomstats/geomstats/blob/master/notebooks/04_practical_methods__from_vector_spaces_to_manifolds.ipynb" target="_blank">from this example.</a></p><p>Correct typo in title</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1810/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>